projects/docs-tools/src/modules/type-definitions/typedoc-adapter.service.ts

```diff
@@ -143,7 +143,7 @@ export class SkyDocsTypeDocAdapterService {
     this.applyCommentTagValues(definition, tags);
 
     const typeParameters = this.getTypeParameterDefinitions(
-      entry.typeParameter
+      entry.typeParameters
```

Comment:
Typedoc renamed this property from `typeParameter` to `typeParameters` - this will fix missing generics currently on the v9 site.